src/aatoolbox/utils/_dates.py

```diff
@@ -0,0 +1,98 @@
+"""Functions for dealing with dates."""
```

Comment:
I was thinking it should as a private module, based on the same link ha! In the accepted answer it says:

> `_...` is often used as 'internal' name. For example, modules starting with `_` shouldn't be used directly; similarly, methods with `_` are supposedly-private and so on. It's just a convention but you should respect it.

So was putting this as a private module. However, see the answer below that you're referring to, so can put this just as `dates.py` because it seems actually unnecessary.